recipes/emsdk/all/conanfile.py

```diff
@@ -29,7 +29,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("nodejs/16.3.0")
+        self.requires("nodejs/[>=20.16.0]")
```

Comment:
please explain the motivation behind this change, thanks!

Reply:
With the older nodeJS version, I got syntax errors when running emcc. After some additional testing, I actually removed the dependency on conan nodejs completely (cf. https://github.com/conan-io/conan-center-index/pull/27213/commits/c39441526d20e106ccbef5ece942c4de34e1e1b0).

Emsdk manages a nodejs installation by itself (4.0.6 even uses 20.18.0, but that wasn't on conan center yet). By removing this dependency, it makes it easier to update the emsdk recipe, and avoids potential clashes between the conan nodejs and the one managed by emsdk.